contributing/code/bc.rst

```diff
@@ -196,21 +234,28 @@ Move to parent class                                Yes
 Add protected property                              Yes
 Remove protected property                           No [7]_
 Reduce visibility                                   No [7]_
+Make public                                         Yes
```

Comment:
Such a change will break child classes that override the formerly protected property: https://3v4l.org/8vbr3